scripts/lint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+if [[ $(flutter format -n .) ]]; then
+    echo "flutter format issue"
```

Comment:
We should echo the error here as well.

Reply:
it's not very relevant because you will just use the fix script to fix right?

this is the script we use in all other repos

Reply:
the flutter format output is pretty useless.